docs/whats-new/dotnet-7-docs.md

```diff
@@ -0,0 +1,295 @@
+- [Generic math](../standard/generics/math.md)
+- [Source generation for platform invokes](../standard/native-interop/pinvoke-source-generation.md)
+- [Source generation for custom marshalling](../standard/native-interop/custom-marshalling-source-generation.md)
+- [Tutorial: Use custom marshallers in source-generated P/Invokes](../standard/native-interop/tutorial-custom-marshaller.md)
+- [Customize a JSON contract](../standard/serialization/system-text-json/custom-contracts.md)
+
+### Updated articles
+
+- [Native AOT Deployment](../core/deploying/native-aot/index.md) - Document AOT analysis warnings
+- [Single-file deployment and executable](../core/deploying/single-file/overview.md) - Update single file incompatibility docs
+- [Prepare .NET libraries for trimming](../core/deploying/trimming/prepare-libraries-for-trimming.md)- Update trimming libraries docs
+- [Trimming options](../core/deploying/trimming/trimming-options.md) - Cleanup trimming options and document TrimMode=full,partial
+- [Tutorial: Containerize a .NET app](../core/docker/build-container.md) - Add `dotnet publish` article for .NET 7 containerization support.
+- [Rate limit an HTTP handler in .NET](../core/extensions/http-ratelimiter.md) - New `RateLimiter` article showing a custom HTTP handler
+- [Configuration providers in .NET](../core/extensions/configuration-providers.md) - Environment variable prefix breaking change
+- [Use HTTP/3 with HttpClient](../core/extensions/httpclient-http3.md) - Update HTTP/3 support for HttpClient in .NET 7
+- [IHttpClientFactory with .NET](../core/extensions/httpclient-factory.md) - New `HttpClient` article
+- [Overview of .NET, MSBuild, and Visual Studio versioning](../core/porting/versioning-sdk-msbuild-vs.md)
+  - Add preview SDK table
+  - Formalize versioning support policy
+- [Overview of the .NET Upgrade Assistant](../core/porting/upgrade-assistant-overview.md) - Merge .NET 7 release branch into main
+- [Upgrade a WCF Server-side Project to use CoreWCF on .NET 6](../core/porting/upgrade-assistant-wcf.md) - Add documentation for CoreWCF extension on Upgrade Assistant
+- [.NET RID Catalog](../core/rid-catalog.md) - Add iOS and Android RIDs
+- [Runtime configuration options for threading](../core/runtime-config/threading.md) - Document a new config switch added to the portable thread pool
+- [MSBuild reference for .NET SDK projects](../core/project-sdk/msbuild-props.md)
+  - Add three new properties
+  - Add docs for project properties that control PDBs and XML publishing
+  - Update the documentation for PublishRelease and PackRelease
+  - Add DisableTransitiveProjectReferences
+  - Cleanup trimming options and document TrimMode=full,partial
+  - Document new PublishRelease and PackRelease properties
+- [How to enable tab completion for the .NET CLI](../core/tools/enable-tab-autocomplete.md) - Merge .NET 7 release branch into main
```

Comment:
Should this be included?  And should it have a different note about the change?

Reply:
Same here, this wasn't updated for .NET 7. It can be removed.